doc/conf.py

```diff
@@ -34,7 +34,7 @@
     subprocess.run(["conda", "list"])
 else:
     print("pip environment:")
-    subprocess.run(["pip", "list"])
+    subprocess.run([sys.executable, "-m", "pip", "list"])
```

Comment:
Is this needed? cc @keewis

Reply:
for the CI this is not necessary, but it might help when building locally. For the difference between `python -m pip` and `pip` see this [blog post](https://snarky.ca/why-you-should-use-python-m-pip/).

Reply:
If you have python2.7 installed alongside pythons 3.6 through 3.9, `pip` will likely default to python 2.7 or python 3.6 unless you're in a virtual environment.  In a virtual environment, there is no difference.  When using system python, with more than one version of python available, this ensures the packages available for import in the current python session are listed.